xcode/Subconscious/Shared/Components/Common/TranscludeView2.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  TranscludeView2.swift
+//  Subconscious
+//
+//  Created by Gordon Brander on 2/22/23.
+//
+
+import SwiftUI
+
+/// A transitional transclude form that does not have a PFP.
+/// We'll replace uses of Transclude2View with TranscludeView once we integrate
+/// pfps throughout the app.
+struct Transclude2View: View {
```

Comment:
nitpick: filename does not match component name (`TranscludeView2` vs `Transclude2View`)

Reply:
Fixed. Thanks!